tests/vmi_configuration_test.go

```diff
@@ -3392,17 +3392,21 @@ func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume
 	return virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
```

Comment:
```
- Return an error instead of using ginkgo assetions making it more predictable.
- Simplify the executed commands string composing
- Get rid of named return values
```

the two former commits have clear motivations. But the third commit is less clear.
- can you state the reason of avoiding `Expect`?
- it's not obviously clear to me why the suggested string manipulation is simpler, or why the new variable names are better
- why is getting rid of named return values good?

I wouldn't mind seeing this commit in its own PR, so the former ones can be quickly merged.

Reply:
>  can you state the reason of avoiding Expect?

Less side effects,  when it return an error we can assert at the test context and make the test easier to understnad (I dont have to get into the function to see how it asserts).

> it's not obviously clear to me why the suggested string manipulation is simpler, 

At first it seem the right thing to do in order to improve readability, looking at this again there is no much difference in readability  using '+' or 'Sprinf'.
I dont have strong opinion on it, I can revert it if you prefer the previous form.

> or why the new variable names are better

Previous variable names were not formatted in the camel-case, for example: 'kvmpitcomm' should be 'kvmpitComm', 'tasksetcmd' should be 'taslsetCmd'.
As for 'args' of the first command, renaming to 'findKvmpitPID' is an attempt to make it easier to understand what it does.
One could say we should have change the second command 'args' variable as well.
In that case there is no much to explain because it add bin-bash prefix where in the previous example it called 'pgrep' to search certain PID .

> why is getting rid of named return values good?

Named return values are declared and initialized with its 'zero' values , in general nothing prevent these variables to get shadowed and introduce unexpected return values.
In this case, the names return values  are redundant because  there is only one return point where it returns values explicitly.
Also, it is not a common practice in our codebase, having it removed make this function style more consistent.

 I think this topic should be taken in account when writing code styling guide for this repo. 
@enp0s3  tagging you because I recall you had an intention to start formalizing  some code styling guidelines for this repo

Reply:
Thanks for your answers. I think they belong in the commit message. Still, for me this last commit is a bit dubious, and I would lgtm this more easily if moved to its own PR, and possibly split to three commits (I now see the danger of named return values, but I don't really find the variable name and string manipulation changes too useful.)

Reply:
I removed this commit for now